sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2831,4 +2831,17 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       }
     }
   }
+
+  test("SPARK-25084: 'distribute by' on multiple columns may lead to codegen issue") {
+    withView("spark_25084") {
+      val count = 1000
```

Comment:
nit: we can probably inline in the next line

Reply:
How to inline? We still use it in the assert.
```
      assert(
        spark.sql(s"select * from spark_25084 distribute by ($distributeExprs)").count()
          === count)
```